components/Layout.tsx

```diff
@@ -15,16 +17,18 @@ const WalletProvider = dynamic(
 export const Layout = ({ children }: any) => {
   return (
     <div className="content">
-      <TopHead />
-      <WalletProvider>
-        <Logo />
-        <section id="dashboard">
-          <Header />
-          <Navbar />
-          <main> {children}</main>
-        </section>
-        <Footer />
-      </WalletProvider>
+      <QueryClientProvider client={queryClient}>
```

Comment:
move along with all other contexts (top-level app)

Reply:
The main page that appears after the root top level page is the layout page
The providers are there for clean code in the root top level page